hack/mock-unittest-data/yarn.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+
+cat << banner-end
+--------------------------------------------------------------------------------
+Generating mock data for yarn unit tests
+--------------------------------------------------------------------------------
+banner-end
+
+tmpdir=$(dirname "$(mktemp --dry-run)")
+
+git clone https://github.com/cachito-testing/cachi2-yarn-berry \
+    --branch=zero-installs \
```

Comment:
I incorporated the changes from https://github.com/cachito-testing/cachi2-yarn-berry/pull/2

Will update if necessary once (if) that is merged